### PR TITLE
CB-11236 We don't need to stop the CM if it is already stopped or una…

### PR DESCRIPTION
**Pull request by topolyai5**
…vailable any other reasons. Like other CM api calls, we retry 5 times before we say that is not running

See detailed description in the commit message.

### CODE REVIEW

File: core/src/main/java/com/sequenceiq/cloudbreak/reactor/handler/cluster/ClusterStopHandler.java [hunk 1]
```diff
@@ -37,7 +42,12 @@ public void accept(Event<ClusterStopRequest> event) {
         ClusterStopResult result;
         try {
             Stack stack = stackService.getByIdWithListsInTransaction(request.getResourceId());
-            apiConnectors.getConnector(stack).stopCluster(false);
+            boolean clusterManagerRunning = apiConnectors.getConnector(stack).clusterStatusService().isClusterManagerRunning();
```
Comment (lacikaaa) [inline core/src/main/java/com/sequenceiq/cloudbreak/reactor/handler/cluster/ClusterStopHandler.java:45]: as I see only `ApiException` is handled here. Are we sure we can't receive any other type of exception when CM is not running?

Comment (topolyai5) [reply core/src/main/java/com/sequenceiq/cloudbreak/reactor/handler/cluster/ClusterStopHandler.java:45]: indeed, I'm checking that

Comment (topolyai5) [reply core/src/main/java/com/sequenceiq/cloudbreak/reactor/handler/cluster/ClusterStopHandler.java:45]: I have stopped the CM and working fine, but I didn't check the exception

Comment (lacikaaa) [reply core/src/main/java/com/sequenceiq/cloudbreak/reactor/handler/cluster/ClusterStopHandler.java:45]: have you tested it with clusterproxy/CCM?

Comment (topolyai5) [reply core/src/main/java/com/sequenceiq/cloudbreak/reactor/handler/cluster/ClusterStopHandler.java:45]: Nit yet, I will try to test it

Comment (topolyai5) [reply core/src/main/java/com/sequenceiq/cloudbreak/reactor/handler/cluster/ClusterStopHandler.java:45]: Answer for the first question:
We can receive an ApiException here, the ApiException store the status code like 502

**Review by lacikaaa — APPROVED** [review]

File: core/src/main/java/com/sequenceiq/cloudbreak/reactor/handler/cluster/ClusterStopHandler.java (hunk 1 above)
Comment (topolyai5) [reply core/src/main/java/com/sequenceiq/cloudbreak/reactor/handler/cluster/ClusterStopHandler.java:45]: Working with cluster proxy as well

Comment (topolyai5) [reply core/src/main/java/com/sequenceiq/cloudbreak/reactor/handler/cluster/ClusterStopHandler.java:45]: dh works as well

**Pull request merged**